### DISCUSSION TITLE
Change look of a Form Submit Button

### DISCUSSION THREAD

**Discussion by ArianYN** [sqlpage/SQLPage#520] [Q&A]
Is it possible to customize the look of a form Submit Button.

Like maybe change the color of it, or apply Narrow to the button and give it an icon instead of text?

**Comment by lovasoa** [comment]
Hello and welcome to SQLPage!

 You can remove the built-in submission button with `'' as validate` and then submit the form with the button component: https://sql.ophir.dev/documentation.sql?component=button#component

```sql
select 
    'form' as component,
    'poem' as id,
    ''     as validate;
select 
    'textarea'     as type,
    'Poem'         as name,
    'Write a poem' as placeholder;
select 
    'button' as component;
select 
    '?action=save' as link,
    'poem'         as form,
    'red'      as color,
    'Save'         as title;
```

**Comment by ArianYN** [comment]
Yes I tried this before , but doing it this way will not place the button inside the Form, resulting in quite some space between the form and the button.

**Comment by lovasoa** [comment]
Yes, the padding below the form was indeed not quite right; this has been fixed and will be in the latest release. In the meantime you can try the main branch on docker with `lovasoa/sqlpage:main` on docker

```
docker run -p 8080:8080 --volume "$(pwd):/var/www" lovasoa/sqlpage:main
```

**Reply by ArianYN** [reply]
Okay, thanks 👍
